src/test/java/org/jfrog/hudson/pipeline/integration/CommonITestsPipeline.java

```diff
@@ -550,15 +550,15 @@ void dockerPullTest(String buildName) throws Exception {
         assertTrue(deps.stream().anyMatch(dep -> dep.getId().equals(imageId)));
     }
 
-    void xrayScanTest(String buildName, boolean failBuild) throws Exception {
+    void xrayScanTest(String buildName, boolean failBuild, boolean printTable) throws Exception {
         Assume.assumeTrue("Skipping Xray tests", JENKINS_XRAY_TEST_ENABLE == null || Boolean.parseBoolean(JENKINS_XRAY_TEST_ENABLE));
-        String str = String.valueOf(failBuild);
-        xrayScanTest(buildName, "xrayScanFailBuild" + str.substring(0, 1).toUpperCase() + str.substring(1), failBuild);
+        String failStr = String.valueOf(failBuild);
+        xrayScanTest(buildName, "xrayScanFailBuild" + StringUtils.capitalize(failStr), failBuild, printTable);
     }
 
-    private void xrayScanTest(String buildName, String pipelineJobName, boolean failBuild) throws Exception {
+    private void xrayScanTest(String buildName, String pipelineJobName, boolean failBuild, boolean printTable) throws Exception {
```

Comment:
To avoid this test failing, in case we decode to change the table titles, I suggest we expose these titles as constants in the build-info. We can String.format for titles which include variables.

Reply:
Im only testing that the table is being print here, not the table contents. I made the headline public so it can be compared.